Applications/FileIO/SWMM/SWMMInterface.h

```diff
@@ -0,0 +1,245 @@
+ /**
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#ifndef SWMMINTERFACE_H_
+#define SWMMINTERFACE_H_
+
+#include <iostream>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <logog/include/logog.hpp>
+
+#include "ThirdParty/SWMMInterface/swmm5_iface.h"
+
+#include "GeoLib/Station.h"
+
+namespace GeoLib {
+    class GEOObjects;
+    class Point;
+    class Polyline;
+}
+
+namespace MeshLib {
+    class Mesh;
+    class Node;
+    class Element;
+}
+
```

Comment:
Better to put the code into a namespace. (Not sure what is the correct name now, but something with IO...)